src/org/omegat/core/Core.java

```diff
@@ -215,11 +214,11 @@ public static IDictionaries getDictionaries() {
     }
 
     public static Segmenter getSegmenter() {
-        return segmenter;
+        return currentProject.getSegmenter();
```

Comment:
`currentProject`  can be null

Reply:
I push the commit which initialize current project in static fields initializer.